src/Employer/Employer.Web/Configuration/EsfaCookieOptions.cs

```diff
@@ -11,5 +12,13 @@ public static class EsfaCookieOptions
             SameSite = SameSiteMode.Strict,
             HttpOnly = true
         };
+
+        public static CookieOptions GetShortLifeHttpCookieOption(IHostingEnvironment env) => new CookieOptions
+        {
+            Secure = !env.IsDevelopment(),
+            SameSite = SameSiteMode.Strict,
+            HttpOnly = true,
+            Expires = DateTimeOffset.UtcNow.AddMinutes(5)
```

Comment:
A user session lasts for longer than 5 minutes, it is likely that the cookies will be gone if the user is distracted for over 5 minutes, have you accounted for if the cookies are not found or considered leaving the cookie for longer duration?

Reply:
I put 5 minutes, which I will speak to PO/BA about tomorrow. The reason I put 5 is because I do not know how long is left of their auth timeout. If the cookies are not found they will see old values of the actual vacancy and cannot submit changes as there are none. This will not block the user form making changes.

Reply:
Make cookie valid only for that session, so it expires if the user session expires?